_docs/ops/maintenance-list.md

```diff
@@ -66,6 +66,10 @@ Review the [detailed guide on customer support]({{site.baseurl}}/docs/ops/custom
 - In [logs.fr.cloud.gov](https://logs.fr.cloud.gov/), go under "Management" -> "Advanced Settings" and check the Kibana [timezone setting](https://www.elastic.co/guide/en/kibana/current/advanced-options.html) (`dateFormat:tz`) - it should be set to `Browser`. If anyone has changed it, change it back to `Browser`.
 - Check for any new CVEs in [Elasticsearch](https://www.cvedetails.com/vulnerability-list/vendor_id-13554/Elasticsearch.html), and [Redis](https://www.cvedetails.com/vulnerability-list/vendor_id-15183/product_id-31837/Pivotal-Software-Redis.html).
 
+## Review expiring certificates
+
+From the jumpbox in each of our four environments, `tooling`, `development`, `staging`, and `production`, run `doomsday dashboard` to review all expiring certificates. Make note of any that are expiring within two weeks and make sure the team is aware.
+
```

Comment:
You can also look at the GUI while on VPN: https://doomsday.fr.cloud.gov/

Reply:
Ah, that's much easier, thank you!  I also see what you mean about Comic Sans... 😆